src/stores/RouteStore.ts

```diff
@@ -12,7 +12,7 @@ export interface RouteStoreState {
 export default class RouteStore extends Store<RouteStoreState> {
     private static getEmptyPath(): Path {
         return {
-            bbox: [0, 0, 0, 0],
+            bbox: undefined,
```

Comment:
We had this discussion before :) In `BasePath` the bbox is declared optional, but here we set it to `[0, 0, 0, 0]` for the empty path. We then check for `route.selectedPath.bbox` here: https://github.com/graphhopper/graphhopper-maps/blob/3d327b00f95ad4d05c8502777045de1b0038de97/src/App.tsx#L53, but this does not what we want. We would have to change this to `route.selectedPath.bbox.every(num => num !== 0)`, but since bbox is declared optional we even need `route.selectedPath.bbox && route.selectedPath.bbox.every(num => num !== 0)`. So we could do this but then I think bbox should not be optional in `BasePath`. Then again if bbox might be missing I think declaring it optional and handling the cases where it is missing is perfectly fine.

Reply:
Yes, I agree. Since [0,0,0,0] is still a valid bbox having it undefined to indicate we really don't have a bbox is probably better.